assets/js/menu/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * A hook controlling a toggleable menu.
+ *
+ * The element should have two children:
+ *
+ *   * one annotated with `data-toggle` being a clickable element
+ *
+ *   * one annotated with `data-content` with menu content
+ */
+const Menu = {
```

Comment:
Ended up with a hook after all. Using `morphdom`'s `onNodeAdded` doesn't work for elements that are created during the initial render and not touched later on. I explored extending it, so that we initialize all the elements on first render and then whenever new elements are added. This could even be abstracted quite nicely, but I felt like it's too much and having a hook probably makes more sense than introducing an entirely separate mechanism for handling JS interaction.